packages/template-react/src/counter/index.js

```diff
@@ -1,29 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
```

Comment:
update component to allow use `--fast-refresh` without problems, if for some reason this component was class based let me know and I can revert this change 👍

Reply:
There's no specific reason. It's just old code ;)